### ISSUE TITLE
Problem in churnsim.py 

### ISSUE THREAD

**Issue by diligejy** [carl24k/fight-churn#25]
Hi, I'm doing data-generation using fight-churn github.

I have a trouble with running churnsim.py
The screen below is an error message that comes out when I run churnsim.py.

How can I fix it?
(p.s I barely ran a churndb.py and created a schema.)

-------------------------------------------------------------------------
run churnsim.py
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
~\Pictures\fight-churn\fightchurn\datagen\churnsim.py in <module>
    246 
    247 
--> 248     run_churn_simulation(model_name, start_date, end_date, init_customers)

~\Pictures\fight-churn\fightchurn\datagen\churnsim.py in run_churn_simulation(model_name, start_date, end_date, init_customers, random_seed)
    232     if random_seed is not None:
    233         random.seed(random_seed) # for random
--> 234     churn_sim = ChurnSimulation(model_name, start_date, end_date, init_customers,random_seed)
    235     churn_sim.run_simulation()
    236 

~\Pictures\fight-churn\fightchurn\datagen\churnsim.py in __init__(self, model, start, end, init_customers, seed)
     45             if version in ('utility','population','country'):
     46                 continue
---> 47             behave_mod=FatTailledBehaviorModel(self.model_name,seed,version)
     48             self.behavior_models[behave_mod.version]=behave_mod
     49             self.model_list.append(behave_mod)

~\Pictures\fight-churn\fightchurn\datagen\behavior.py in __init__(self, name, random_seed, version)
    134         self.exp_fun = lambda x: np.power(self.exp_base,x)
    135 
--> 136         super(FatTailledBehaviorModel,self).__init__(name,random_seed,version)
    137 
    138     def scale_correlation_to_covariance(self):

~\Pictures\fight-churn\fightchurn\datagen\behavior.py in __init__(self, name, random_seed, version)
     91 
     92         # Save to a csv
---> 93         save_path = os.getenv('CHURN_OUT_DIR') + name + '/'
     94         os.makedirs(save_path, exist_ok=True)
     95         copy_path = save_path + name + '_simulation_model.csv'

TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'

**Comment by carl24k** [comment]
Hi, it looks like you need to set the environment variable CHURN_OUT_DIR. Do you have that set? Definitely try setting it along with the other environment variables.  I think it should have been defaulted for you, but maybe the default initialization has a mistake (I will check that out...)

**Comment by diligejy** [comment]
Hi.

I've tried in churn_db_sim.ipynb with adding environment variable

originally 
There are 3 variables in churn_db_sim.ipynb
```
os.environ['CHURN_DB']='churn'
os.environ['CHURN_DB_USER']='postgres'
os.environ['CHURN_DB_PASS']='diligejy'
```

However as you said, we need `CHURN_OUT_DB`
so I added like this. `os.environ['CHURN_OUT_DIR']="C:/Users/diligejy/Pictures/fight-churn/output"`

I don't know what means `CHURN_OUT_DIR`. 
So I can't ensure path is proper for your intention.

After executing churnsim.py, new folder and file comes up like below capture
![image](https://user-images.githubusercontent.com/40735154/135798357-2e02bc09-2d34-4147-bdba-ebd6890b1eab.png)


First of all, I think it works, so I'll try more and ask if it doesn't work.

Thanks.
